docs/api-endpoints.md

```diff
@@ -5,6 +5,20 @@
 - Team admin can request for an API Account through CSS App for managing gold integrations. Using the API Account credentials, the team admin can request for a token with an expiry duration
 - Token url would be `https://loginproxy.gov.bc.ca/auth/realms/standard/protocol/openid-connect/token`
 
+### Requesting a token
+
+- Request a token using below command
+
+  ```sh
+  export CLIENT_ID=
+  export CLIENT_SECRET=
+  export TOKEN_URL=
+
+  curl -d grant_type=client_credentials -d client_id=$CLIENT_ID -d client_secret=$CLIENT_SECRET $TOKEN_URL
```

Comment:
looking good;
I believe `curl` implicitly makes `POST` method with data options (-d), I would put `-X POST` to let them know it is `POST` method explicitly, and could pipe the data to `jq` to extract that specific attribute from the data.

so it can be updated something like:
`curl -X POST -d grant_type=client_credentials -d client_id=$CLIENT_ID -d client_secret=$CLIENT_SECRET $TOKEN_URL | jq .access_token`

Reply:
I might remove `jq` as many people may not have it installed, so its upto them how to extract the token from the response